docs/measurement.rst

```diff
@@ -0,0 +1,169 @@
+.. _rstmeasurement:
+
+===========
+Measurement
+===========
+
+The measurement element is used to record individual measurements. It can capture
+quantitative, ordinal (e.g., absent/present), or categorical measurements.
+
+
+**Data model**
+
+
+.. list-table:: Definition  of the ``Quantity`` element
+   :widths: 25 25 50 50
+   :header-rows: 1
+
+   * - Field
+     - Type
+     - Status
+     - Description
+   * - description
+     - string
+     - optional
+     - free text.
+   * - assay
+     - OntologyClass
+     - required
```

Comment:
This is a problem - we can't _require_ a value for a measurement defined as a complex quantity. Ideally this would be a onof Value or ComplexQuantity